charts/cluster-proportional-autoscaler/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: cluster-proportional-autoscaler
-version: 1.1.0
+version: 1.1.1
```

Comment:
Less certain about helm chart version here - I would expect a minor version bump for CPA itself as this is introducing a new feature.

Should we probably bump this later once we have a new CPA version released?

Reply:
Yes we can bump the version as a separate PR for the next release.